### ISSUE TITLE
!secret no longer works in apps.yaml

### ISSUE THREAD

**Issue by nickneos** [hassio-addons/addon-appdaemon3#39]
# Problem/Motivation

having !secret in apps.yaml no longer works

## Expected behavior
!secret is parsed in apps.yaml as was the case prior to last release

## Actual behavior

```
2019-04-02 22:19:17.599568 INFO API is disabled
2019-04-02 22:19:17.629602 INFO AppDaemon: HASS: Connected to Home Assistant 0.90.2
2019-04-02 22:19:18.014390 INFO AppDaemon: Got initial state from namespace default
2019-04-02 22:19:22.822573 INFO AppDaemon: Reading config
2019-04-02 22:19:23.217004 WARNING AppDaemon: Error loading configuration
2019-04-02 22:19:23.217780 WARNING AppDaemon: parser says
2019-04-02 22:19:23.218490 WARNING AppDaemon:   in "<unicode string>", line 79, column 15:
          gw_mac: !secret xiaomi_mac
                  ^
2019-04-02 22:19:23.218930 WARNING AppDaemon: could not determine a constructor for the tag '!secret'
2019-04-02 22:19:23.220558 WARNING AppDaemon: File '/config/appdaemon/apps/apps.yaml' invalid structure - ignoring

```
## Steps to reproduce

have a !secret reference in apps.yaml

**Comment by Luke15153** [comment]
Perhaps it is related to this issue?
https://github.com/home-assistant/appdaemon/issues/538#issuecomment-478983353

**Comment by frenck** [comment]
This add-on does only install AppDaemon:

<https://github.com/hassio-addons/addon-appdaemon3/blob/master/appdaemon/requirements.txt>

Dependencies of AppDaemon are not locked in by the add-on, but are defined in:

<https://github.com/home-assistant/appdaemon/blob/dev/setup.py#L18>

Since AppDaemon does not lock in a version, the latest PyYaml gets installed, which is apparently broken.

Sorry, but there is nothing I can do. As soon as AppDaemon releases a working version (by doing proper version pinning), I'll release an update as soon as possible.



<blockquote><img src="https://avatars1.githubusercontent.com/u/30772201?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/hassio-addons/addon-appdaemon3">hassio-addons/addon-appdaemon3</a></strong></div><div>AppDaemon3 - Community Hass.io Add-on for Home Assistant - hassio-addons/addon-appdaemon3</div></blockquote>
<blockquote><img src="https://avatars1.githubusercontent.com/u/13844975?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/home-assistant/appdaemon">home-assistant/appdaemon</a></strong></div><div>:page_facing_up: Python Apps for Home Automation. Contribute to home-assistant/appdaemon development by creating an account on GitHub.</div></blockquote>

**Comment by nickneos** [comment]
no worries...thanks

**Comment by frenck** [comment]
v2.0.2 of the add-on has just been released, which upgrades AppDaemon to v3.0.3, which addresses the secrets issue.
